src/main/java/com/google/sps/datastore/GetDatastore.java

```diff
@@ -0,0 +1,74 @@
+package com.google.sps.datastore;
+
+import java.util.ArrayList;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+
+//PCAP storage class
+public class GetDatastore {
+
+    public ArrayList<PCAPdata> dataTable = new ArrayList<PCAPdata>();
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
These two should go in the constructor even though it would not matter much but for readability.
And they should be private.

Reply:
removed dataTable, ran into issues moving datastore into constructor, switch it to private